test/utils/docker.js

```diff
@@ -0,0 +1,48 @@
+const childProcess = require('child_process');
+const path = require('path');
```

Comment:
Hmm, I don't think I agree. As you know, I am not a JS expert, but the functionality seems fairly clear to me (the functions are small and everything is semantically-named). There might be more in here than strictly necessary, but given the urgency and high priority of the task, and the small size of the file, that seems OK to me 🤔 

Moreover, this file, as a utility, is secondary to the aims of this repository—it's possible that, just as this replaces an existing but functional solution, this might go away, in which case the extra effort for elaborate documentation would be a waste of resources. More documentation might be helpful for improving one's understanding of node, but that is not the purpose of this file.

However, I'm happy to revert to the mosaic team for additional information if something is particularly problematically unclear.

Reply:
Fair enough. From the first look I wasn't sure how this file (docker.js + travis) integrates together. Understood it after you explained below:
```
This approach does rely on Travis's docker support. That's why docker is not specifically installed and the docker.js file is able to use the docker-compose command.
```
May be a line a two on that part will be helpful.  But if you think it's taking time we can do later.

Reply:
Ah, yeah, that's a super good point!

It took me a non-trivial amount of time to understand why I was not seeing docker installation in the travis output. I will document that implicit dependency 🙌

Reply:
Done